### PR TITLE
Updated the descriptions of the live and batch jobs

### PR DESCRIPTION
**Pull request by mt-empty**
## Proposed changes

Updated the descriptions of the live and batch jobs on their respective pages and the main page.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] DevOps related

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/components/mainPage/JobSelectionSection.jsx [hunk 1]
```diff
@@ -38,13 +38,13 @@ const JobSelectionSection = forwardRef((props, ref) => {
           <JobButton
             icon={LiveIcon}
             jobType="LIVE JOB"
-            description="See results in real time. Up to 10 images and videos"
+            description="Submit up to 10 images and videos and watch the results appear in real time"
```
Comment (idomo1) [inline src/components/mainPage/JobSelectionSection.jsx:41]: Not a fan of the wording of this

Comment (mt-empty) [reply src/components/mainPage/JobSelectionSection.jsx:41]: feel free to suggest any

Comment (idomo1) [reply src/components/mainPage/JobSelectionSection.jsx:41]: How about: "Submit up to 10 images and videos and see the results in real time"?

Comment (mt-empty) [reply src/components/mainPage/JobSelectionSection.jsx:41]: `see` doesn't sound good, how about `Submit up to 10 images and videos and watch the results in real time` 

```
To see means to be aware of what is around you by using your eyes.

For example: "I can see the smoke from here."

To watch means to deliberately look at something for a period of time, especially something that is changing or moving.

For example: "I watched the cricket."
```

Comment (idomo1) [reply src/components/mainPage/JobSelectionSection.jsx:41]: I don't think `watch` sounds good

Comment (Akshith-Patil) [reply src/components/mainPage/JobSelectionSection.jsx:41]: What about `"...view the results in real time"?`

**Review by Akshith-Patil — APPROVED** [review]

**Pull request merged**